README.md

```diff
@@ -24,7 +24,9 @@ $ go get github.com/matryer/moq
 ### Usage
 
 ```
-moq [flags] destination interface [interface2 [interface3 [...]]]
+moq [flags] source-dir interface [interface2 [interface3 [...]]]
```

Comment:
I never thought, that `destination` is a good term for the directory, where the source code (definition) of the interface is located that should be mocked. `source-dir` is better, but I feel like it is still not 100% clear/obvious. Maybe an additional sentence in the `README.md` is needed to explain the the correct meaning of `source-dir` (especially, because it needs to be a path to a directory and not the import statement for a Go package).

Reply:
Added note to `README.md` as suggested.